versioning as configurable option (#107)
delvedor/find-my-way: README.md

})
```

<a name="custom-versioning"></a>
By default `find-my-way` uses [accept-version](./lib/accept-version.js) strategy to match requests with different versions of the handlers. The matching logic of that strategy is explained [below](#semver). It is possible to define the alternative strategy:
```js
const customVersioning = {
  // storage factory
  storage: function () {
    let versions = {}
    return {
      get: (version) => { return versions[version] || null },
      set: (version, store) => { versions[version] = store },
      del: (version) => { delete versions[version] },
      empty: () => { versions = {} }
    }
  },
  deriveVersion: (req, ctx) => {
    return req.headers['accept']
  }
}

const router = FindMyWay({ versioning: customVersioning });
```

> Custom strategy object should contain next properties:

Reviewer: ```suggestion
The custom strategy object should contain next properties:
```

> * `storage` - factory function for the storage, where different handler versions will be stored.

Reviewer: ```suggestion
* `storage` - the factory function for the Storage of the handlers based on their version.
```

Also you should probably add a link to the default one here .

> * `deriveVersion` - function which determines the version based on the request

Reviewer: ```suggestion
* `deriveVersion` - the function to determine the version based on the request
```

> Signature of the functions and objects must match the one from the example above.

Reviewer: ```suggestion
The signature of the functions and objects must match the one from the example above.
```
